react-forms/src/Select.tsx

```diff
@@ -44,10 +44,28 @@ export default function Select<T>(props: IProps<T>) {
 	const [guid, setGuid] = React.useState<string>("");
 	const [showHelp, setShowHelp] = React.useState<boolean>(false);
 	
-	 React.useEffect(() => {
-		setGuid(CreateGuid());
-	  }, []);
+  React.useEffect(() => {
+    setGuid(CreateGuid());
+  }, []);
+
+  React.useEffect(() => {
```

Comment:
you may want to do a `console.warn("Value is not in options")` in this case to make sure this get's caught since it is bad design